google-account-plugin/src/com/google/cloud/tools/intellij/stats/GoogleUsageTracker.java

```diff
@@ -81,17 +81,15 @@ public void trackEvent(@NotNull String eventCategory,
         List<BasicNameValuePair> postData = Lists.newArrayList(analyticsBaseData);
         postData.add(new BasicNameValuePair("tid", analyticsId));
 
-        postData.add(new BasicNameValuePair("cd19", eventCategory));  // Event type
```

Comment:
I mean we can do that with your new design, but not with this interface to be backward-compatible.